azurelinuxagent/common/utils/restutil.py

```diff
@@ -170,6 +174,12 @@ def _get_http_proxy(secure=False):
     return host, port
 
 
+def cleanup_sas_tokens_from_urls(url):
```

Comment:
Preferred name: `redact_sas_tokens_in_urls`. The word "cleanup" could mean many things, but the intent is to do exactly this one thing.